tests/test-sessions/test-sessions-common/src/main/java/org/eclipse/jetty/server/session/TestServer.java

```diff
@@ -93,6 +94,11 @@ public SessionHandler newSessionHandler()
         return h;
     }
 
+    public ServerConnector getConnector()
+    {
+        return _server.getBean(ServerConnector.class);
+    }
```

Comment:
Because we return a `ServerConnector` and not a `Connector`, I think this method should be named `getServerConnector()`.
However, it's in a test class, so I can live with just `getConnector()`.